test/e2e/clusterctl_upgrade_test.go

```diff
@@ -37,12 +37,12 @@ var (
 	providerDockerPrefix  = "docker:v%s"
 )
 
-var _ = Describe("When testing clusterctl upgrades (v0.3=>v1.5=>current)", func() {
```

Comment:
I think this needs a rebase (according to `make lint` we have broken code in this file)